modules/power-monitoring-deploying-kepler-redfish.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/installing-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deploying-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure the {PM-kepler} to use the Redfish as the source for platform power.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*.
+
+. Create a {PM-kepler} instance with `.spec.exporter.redfish` specifying the `secretRef`. `secretRef` refers to the name of secret which contains credentials to initialize `RedfishClient`.
++
+[NOTE]
+====
+The `openshift-power-monitoring` namespace is created after the {PM-kepler} is deployed.
+====
+
+. Create the secret under `openshift-power-monitoring` namespace.
++
+[source,terminal]
+----
+$ oc -n openshift-power-monitoring create secret generic redfish-secret --from-file=redfish.csv
```

Comment:
we should mention  that 
- the secret must be `Opaque` type
- the secret key must be `redfish.csv`